update-third-party-libs.sh

```diff
@@ -14,7 +14,7 @@
 
 (cd both/components/; wget -N http://www.jflei.com/jChester/jChester.js)
 
-(cd both/components/; curl -s https://raw.githubusercontent.com/OpenBookPrices/country-data/master/data/countries.json | sed -e '1s/^/countries = /' -e '$a\;' > countries.js)
+(cd both/components/; curl -s https://raw.githubusercontent.com/OpenBookPrices/country-data/master/data/countries.json | sed -e '1s/^/JSON.stringify(/' -e '$a\,function(k,v,u){if(k!=="name"&&k!=="alpha2"&&isNaN(k)){return u}return v},2)' | node -p | sed -e '1s/^/countries = /' -e '$a\;' > countries.js)
```

Comment:
What's `u` here? The docs on this replacer method say that the function only takes 2 arguments: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Using_native_JSON#The_replacer_parameter.

I think a ternary operator would be cleaner here:

```javascript
return (k!=="name"&&k!=="alpha2"&&isNaN(k)) ? u : v
```

Why do you need to check `isNaN(k)`?